astropy/convolution/convolve.py

```diff
@@ -20,17 +20,16 @@
 from ..modeling.core import _CompoundModelMeta
 from .utils import KernelSizeError, has_even_axis, raise_even_kernel_exception
 
-# Turn the faulthandler ON to help catch any signals, e.g. segfaults
-# or asserts. This doesn't, currently, work with Jupyter Notebook.
-faulthandler.enable()
-
 # Find and load C convolution library
-lib_path = glob.glob(os.path.join(os.path.dirname(__file__), 'lib_convolve*'))[0]
-if sys.platform.startswith('win'):
-    libConvolve = ctypes.windll.LoadLibrary(lib_path)
+lib_paths = glob.glob(os.path.join(os.path.dirname(__file__), 'lib_convolve*'))
+if len(lib_paths) > 0:
+    lib_path = lib_paths[0]
```

Comment:
I don't see these being used anywhere else after import. Maybe clean them up when they are not needed anymore? Or maybe it is not necessary? Just a (sleep-deprived) thought.

```python
del lib_path
del lib_paths
```

Reply:
Fixed